editor/src/core/tailwind/tailwind-options.tsx

```diff
@@ -298,6 +299,11 @@ export function useGetSelectedClasses(): {
     'ClassNameSelect elementPaths',
   )
 
+  const allElementProps = useEditorState(
+    (store) => store.editor.allElementProps,
```

Comment:
`useEditorState` runs a shallow equality by default, if allElementProps is large, this will iterate over all the keys. maybe it's worth providing a strict equality check here? I'm not 100% sure about the tradeoff

Reply:
That _sounds_ like something we should be doing in general for `useEditorState` if we're doing a deep equality pass?